configure.ac

```diff
@@ -220,6 +220,22 @@ AC_ARG_ENABLE([fast-sdk],
 AC_MSG_RESULT([$enable_fast_sdk])
 TS_ARG_ENABLE_VAR([use], [fast-sdk])
 
+# Curl support for traffic_top
+AC_MSG_CHECKING([whether to enable CURL])
+AC_ARG_ENABLE([curl],
+  [AS_HELP_STRING([--disable-curl],[turn off CURL support for traffic_top])],
+  [],
+  [enable_curl=yes]
+)
+AC_MSG_RESULT([$enable_curl])
+
+if test "x${enable_curl}" = "xyes"; then
+AX_LIB_CURL([7.19], [has_curl=1],[has_curl=0])
+AM_CONDITIONAL([HAS_CURL], [test "x$has_curl" = "xyes"])
+else
+AM_CONDITIONAL([HAS_CURL], [false])
+fi
```

Comment:
Yes, that worked.  I tried replacing lines 233 to 234 with using the AC_DEFINE yesterday and I was getting:
configure: error: conditional "HAS_CURL" was never defined.